### ISSUE TITLE
Foreign stocks key.stats not working although stock.prices are?

### ISSUE THREAD

**Issue by phlsmk** [business-science/tidyquant#41]
Hi, thanks for the great library. I'm doing some work with foreign stocks and though I've seen that people have had issues with key.ratios vs key.stats, etc (due to different ticker conventions between data sources Yahoo vs Morningstar), I seem to be having issues retrieving stock data even with the same source (Yahoo Finance).

Take FMG (Australian exchange) for instance, stock prices seem OK:

> tq_get("FMG.AX", get = "stock.prices", from = "2017-05-01")
A tibble: 23 × 7
         date  open  high   low close   volume adjusted
       <date> <dbl> <dbl> <dbl> <dbl>    <dbl>    <dbl>
1  2017-05-01  5.26 5.330 5.160  5.25 13088483     5.25
2  2017-05-02  5.25 5.350 5.230  5.27 18140460     5.27

But I can't get most key.stats like shares outstanding?

> tq_get("FMG.AX", get="key.stats",complete_cases=FALSE) %>% select(Shares.Outstanding)
A tibble: 1 × 1
  Shares.Outstanding
               <dbl>
1                 NA

I understand both of these should be coming from Yahoo Finance. 

Thanks much!

**Comment by phlsmk** [comment]
Just taking a glance, could this be in part because you're removing punctuation that may be necessary for foreign tickers in the key.stats routine but not stock.prices?

        stringr::str_replace_all("[[:punct:]]", "")

**Comment by mdancho84** [comment]
Yep, this might be the issue. Let me take a look and see what I can come up with. Thanks for pointing it out.
